fhir-search/src/main/java/com/ibm/fhir/search/exception/SearchExceptionUtil.java

```diff
@@ -49,9 +50,22 @@ public static FHIRSearchException buildNewParseParameterException(final String n
         return new FHIRSearchException(msg, e).withIssue(ooi);
     }
 
+    /**
+     * creates a new parse parameters exception
+     *
+     * @param name
+     * @param e
+     * @return
+     */
+    public static FHIRSearchException buildNewParseParametersException(Exception e) {
+        String msg = String.format(PARSE_PARAMETERS_EXCEPTION);
+        OperationOutcome.Issue ooi = FHIRUtil.buildOperationOutcomeIssue(msg, IssueType.INVALID);
+        return new FHIRSearchException(msg, e).withIssue(ooi);
```

Comment:
Line 20,21 are almost identical, I'd consider pumping in more information in this OperationOutcome so that it's more specific to the failing reason.

Reply:
For this exception, it's only going to be due to config-related exceptions, which will be in the "cause" exception. That seemed consistent with the existing behavior of buildNewParseParameterException.

Reply:
Sounds good